native/src/components/PageDetail.tsx

```diff
@@ -33,13 +39,25 @@ type PropsType = {
 
 const PageDetail: React.FC<PropsType> = (props: PropsType): ReactElement => {
   const { identifier, information, theme, language, onLinkClick, linkLabel } = props
+  const { t } = useTranslation('pois')
+
+  const navigationUrl = Platform.select({
+    ios: `maps:0,0?q=${information}`,
+    android: `geo:0,0?q=${information}`
+  })
```

Comment:
TBH I don't think this should be part of a generic PageDetail component but instead be moved to a separate component. Then you can also omit the wrapping in a fragment and checking for `linkLabel` and `onLinkClick` props below as those can be required in a separate component.

Reply:
hm so what do you prefer? passing the whole component? or sth like header and body part inside of the generic component.

Reply:
Just split it in two components (one PageDetail as it was before and a separate one for the map deep linking and then just render both separately.